diff --git a/build/fork/useLinking.native.js b/build/fork/useLinking.native.js
index c080a5853fa71f2708c6ed0f15092b6f43231f76..0a00c6697e0f17020e0bdfac6b39d62b6b51e51c 100644
--- a/build/fork/useLinking.native.js
+++ b/build/fork/useLinking.native.js
@@ -122,16 +122,9 @@ function useLinking(ref, { enabled = true, prefixes, filter, config, getInitialU
                 if (typeof url !== 'string') {
                     return url.then((url) => {
                         const state = getStateFromURL(url);
-                        if (typeof url === 'string') {
-                            // If the link were handled, it gets cleared in NavigationContainer
-                            onUnhandledLinking((0, extractPathFromURL_1.extractExpoPathFromURL)(prefixes, url));
-                        }
                         return state;
                     });
                 }
-                else {
-                    onUnhandledLinking((0, extractPathFromURL_1.extractExpoPathFromURL)(prefixes, url));
-                }
             }
             state = getStateFromURL(url);
         }
@@ -144,7 +137,7 @@ function useLinking(ref, { enabled = true, prefixes, filter, config, getInitialU
             },
         };
         return thenable;
-    }, [getStateFromURL, onUnhandledLinking, prefixes]);
+    }, [getStateFromURL]);
     (0, react_1.useEffect)(() => {
         const listener = (url) => {
             if (!enabled) {
